### PR TITLE
Change fonts

### PR DESCRIPTION
**Pull request by jkosir**
https://celtra.atlassian.net/browse/MAB-13232 dependency

### CODE REVIEW

**Comment by jkosir** [comment]
@zan-kusterle How do we regenerate image snapshots for tests?

**Comment by zan-kusterle** [comment]
@jkosir `docker-compose run test -u`.

File: src/stateless/typography.less [hunk 1]
```diff
@@ -1,30 +1,51 @@
...
-    src: url("../assets/fonts/SF-UI-Display-Regular.woff") format("woff");
-}
+/* Semilight */
+@font-face {font-family: 'Segoe UI map-weights'; font-weight: 300; src: local("Segoe UI Semilight"), local('SegoeUI-Semilight');}
+/* Regular */
+@font-face {font-family: 'Segoe UI map-weights'; font-weight: 400; src: local("Segoe UI");}
+/* Medium */
+@font-face {font-family: 'Segoe UI map-weights'; font-weight: 500; src: local("Segoe UI Semibold"), local('SegoeUI-Semibold');}
+/* Bold */
+@font-face {font-family: 'Segoe UI map-weights'; font-weight: 700; src: local("Segoe UI Bold"), local('SegoeUI-Bold');}
```
Comment (zigomir) [inline src/stateless/typography.less]: Minor: mixing single and double quotes.

**Comment by jkosir** [comment]
@zan-kusterle After #70 was merged test are failing on master, any idea what's happening?

**Pull request closed without merging**